src/components/table/table-body-cell/table-body-cell.styles.ts

```diff
@@ -0,0 +1,8 @@
+import { withStyle } from 'baseui';
+import { StyledTableBodyCell } from 'baseui/table-semantic';
+
+export const styled = {
```

Comment:
To be clear, did you isolate these styled components because we are using them in table-virtualized, or is there any other reason?

Reply:
Correct